Tests/ArgumentParserUnitTests/DumpHelpGenerationTests.swift

```diff
@@ -59,15 +59,73 @@ extension DumpHelpGenerationTests {
     @OptionGroup(title: "Other")
     var options: Options
   }
-  
+
+  struct C: ParsableCommand {
```

Comment:
question: Should there be a test that covers default values of enumerable option too?

Reply:
Indeed - this case should be covered by the `defaultColor` property in this command! :) Was there a more robust test case you were thinking of that I missed?